From 42b7f04ae742bcd9d39b8d6f8281d6833263061a Mon Sep 17 00:00:00 2001
From: Richard Palethorpe <rpalethorpe@suse.com>
Date: Mon, 25 Jul 2022 10:28:47 +0100
Subject: [PATCH 3/4] read_all: Allow /sys/power/wakeup_count

This was filtered because it could block at certain times. However we
will now kill the worker and continue.

This could still result in a TBROK test failure if the process can't
be killed (quick enough). Possibly this can be considered a kernel bug
because reading this file should not make an unprivileged process
unkillable. Or else the tester needs to increase LTP_RUNTIME_MUL.

Signed-off-by: Richard Palethorpe <rpalethorpe@suse.com>
Reviewed-by: Li Wang <liwang@redhat.com>
---
 testcases/kernel/fs/read_all/read_all.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/testcases/kernel/fs/read_all/read_all.c b/testcases/kernel/fs/read_all/read_all.c
index 02c8f1754..c36203846 100644
--- a/testcases/kernel/fs/read_all/read_all.c
+++ b/testcases/kernel/fs/read_all/read_all.c
@@ -95,7 +95,6 @@ static int timeout_warnings_left = 15;
 
 static char *blacklist[] = {
 	NULL, /* reserved for -e parameter */
-	"/sys/power/wakeup_count",
 	"/sys/kernel/debug/*",
 	"/sys/devices/platform/*/eeprom",
 	"/sys/devices/platform/*/nvmem",
-- 
2.35.3

